pkg/cmd/cli/power/init.go

```diff
@@ -0,0 +1,35 @@
+/*
+
+ MIT License
+
+ (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+
+ Permission is hereby granted, free of charge, to any person obtaining a
+ copy of this software and associated documentation files (the "Software"),
+ to deal in the Software without restriction, including without limitation
+ the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ and/or sell copies of the Software, and to permit persons to whom the
+ Software is furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included
+ in all copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+ OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ OTHER DEALINGS IN THE SOFTWARE.
+
+*/
+
+package power
+
+func init() {
+	powerOffCmd.Flags().Bool("force", false, "Force power off a node")                        // ForceOff
+	powerOffCmd.Flags().Bool("button", false, "Power off a node by pushing the power button") // PushPowerButton
+	powerOffCmd.Flags().Bool("nmi", false, "Power off a node to trigger a crash/core dump")   // Nmi
+	powerOffCmd.Flags().Bool("restart", false, "Restart a node")                              // ForceRestart
+	powerOnCmd.Flags().Bool("force", false, "Force power on a node")                          // ForceOn
```

Comment:
What is a "force" power on/off?

Reply:
Not sure exactly.  I added it since it was in the lib: https://github.com/stmcginnis/gofish/blob/main/redfish/computersystem.go#L392-L411

How hard can one push the power button to turn it on?  Does it add more electricity? 😄

Reply:
lol.

Well, I'm more concerned with the UX side of this. I can already see some questions we'll get pretty quick from users if we don't add some of our own clarity.

I have a commit incoming that rejiggers some of this.